File: BleDemo/app/src/main/java/com/example/bledemo/MainActivity/MainActivity.java
package com.example.bledemo.MainActivity;

import androidx.appcompat.app.AppCompatActivity;
import androidx.recyclerview.widget.RecyclerView;

import android.os.Bundle;
import android.widget.TextView;

import com.example.bledemo.Bluetooth.BluetoothManager;
import com.example.bledemo.Bluetooth.DeviceData;
import com.example.bledemo.R;
import com.google.android.material.bottomnavigation.BottomNavigationItemView;

import java.util.ArrayList;

public class MainActivity extends AppCompatActivity {
    TextView TextView_BluetoothName;
    BottomNavigationItemView BottomNavigationView_SelectButton;
    BluetoothManager bluetoothManager;

    private ArrayList<DeviceData> arrayList;
    private MainAdapter mainAdapter;
    private RecyclerView recyclerView;
    private int e;


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        bluetoothManager = new BluetoothManager(getApplicationContext());


    }
}

File: TextView/app/src/main/java/com/example/textview/MainActivity.kt
package com.example.textview

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.activity.result.contract.ActivityResultContracts
import com.example.textview.databinding.ActivityMainBinding


class MainActivity : AppCompatActivity() {
    //전역 변수로 바인딩 객체 선언
    private var mBinding: ActivityMainBinding?= null
    //매번 null 체크를 할 필요 없이 편의성을 위해 바인딩 변수 재 선언
    private val binding get() = mBinding!!


    override fun onCreate(savedInstanceState: Bundle?) {  //앱이 최초 실행됐을때 수행.
        super.onCreate(savedInstanceState)
        mBinding = ActivityMainBinding.inflate(layoutInflater)

        setContentView(binding.root)

        binding.tvTitle.setText("안녕하세욧 도로롱입니다.")

    }

    override fun onDestroy() {
        mBinding = null
        super.onDestroy()

    }
}